server/rpc/query/QueryHandler.java

```diff
@@ -90,6 +90,7 @@ public void handleRequest(Transaction.Req request) {
     }
 
     private void match(Transaction.Req request, QueryProto.Query.Match.Req req, Options.Query options) {
+        if (!options.prefetch().isPresent()) options.prefetch(true);
```

Comment:
I feel like these default values should not be defined "scattered" across this class. I feel like they should be aggregated somewhere as a set of enums or constants of some sorts, which sits in a class that holds the relevant domain knowledge.